src/Persistence/ResetPasswordRequestRepositoryInterface.php

```diff
@@ -6,20 +6,48 @@
 
 /**
  * @author Jesse Rushlow <jr@rushlow.dev>
- * @author Ryan Weaver <weaverryan@gmail.com>
+ * @author Ryan Weaver   <ryan@symfonycasts.com>
  */
 interface ResetPasswordRequestRepositoryInterface
 {
+    /**
+     * Create a new ResetPasswordRequest object.
+     *
+     * @param object $user        User entity - typically implements Symfony\Component\Security\Core\User\UserInterface
+     * @param string $selector    A non-hashed random string used to fetch a request from persistence
+     * @param string $hashedToken The hashed token used to verify a reset request
+     */
     public function createResetPasswordRequest(object $user, \DateTimeInterface $expiresAt, string $selector, string $hashedToken): ResetPasswordRequestInterface;
 
+    /**
+     * Get the unique user entity identifier from persistence.
+     *
+     * @param object $user User entity - typically implements Symfony\Component\Security\Core\User\UserInterface
+     */
     public function getUserIdentifier(object $user): string;
 
+    /**
+     * Save a reset password request entity to persistence.
+     */
     public function persistResetPasswordRequest(ResetPasswordRequestInterface $resetPasswordRequest): void;
 
+    /**
+     * Get a reset password request entity from persistence, if one exists, using the request's selector.
+     *
+     * @param string $selector A non-hashed random string used to fetch a request from persistence
+     */
     public function findResetPasswordRequest(string $selector): ?ResetPasswordRequestInterface;
 
+    /**
+     * Get a users most recent reset password request that is not expired, if one exists.
+     *
```

Comment:
Returns the date - this makes it sound like it returns the reset password request itself.

Reply:
good catch!

done